.github/workflows/check-changelog.yml

```diff
@@ -0,0 +1,34 @@
+name: Check changelog
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - 'release/**'
+jobs:
+  check-changelog:
+    name:    Check changelog
+    runs-on: ubuntu-20.04
+    steps:
+      # clone the repository
+      - uses: actions/checkout@v2
+      # enable dependencies caching
+      - name: Add composer to cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/composer/
+          key:  ${{ runner.os }}-composer-${{ hashFiles('composer.lock') }}
+      # setup PHP, but without debug extensions for reasonable performance
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          tools:    composer
+          coverage: none
+      # Install composer packages.
+      - run: composer self-update 2.0.6 && composer install --no-progress
```

Comment:
Maybe you could add -`-no-interaction` flag to prevent issue when composer is asking a user's input.

Reply:
I am following the existing patterns like the following so I guess it should be fine.

https://github.com/Automattic/woocommerce-payments/blob/cf41b6e01bb74d5a02c0de290726237d24a35d2b/.github/workflows/e2e-test.yml#L115-L115

https://github.com/Automattic/woocommerce-payments/blob/cf41b6e01bb74d5a02c0de290726237d24a35d2b/.github/workflows/php-lint-test.yml#L30-L30